src/sentry/static/sentry/app/sentryTypes.jsx

```diff
@@ -311,6 +326,8 @@ let SentryTypes = {
   Tag,
   PageLinks,
   Project,
+  Series,
+  SeriesUnit,
```

Comment:
Will this be used anywhere?

Reply:
Yeah I think other charts will need to use it

Reply:
Series is used in basechart and seriesunit will be used later by other charts, but is used currently only inside series.